bin/melm

```diff
@@ -668,6 +676,14 @@ if ($do_assignment) {
     }
 }
 
+#Push on a temp file if we are running a sequence from the command line
+if (defined $cli_seq){
```

Comment:
An undefined scalar is false `if ($cli_seq)` is cleaner. Same goes for above. Also really don't like the way this is a temp file. That can be done easily enough through <( echo -e '>\nsequencehere'). A better plan is to just create a sequence object direct from the parameter.